src/plugins/controls/public/range_slider/embeddable/range_slider_embeddable.tsx

```diff
@@ -397,11 +367,8 @@ export class RangeSliderEmbeddable
         searchSource.setField('query', query);
       }
 
-      const {
-        rawResponse: {
-          hits: { total },
-        },
-      } = await lastValueFrom(searchSource.fetch$());
+      const resp = await lastValueFrom(searchSource.fetch$());
+      const total = get(resp, 'rawResponse.hits.total');
```

Comment:
nit: I would avoid using get here. Firstly, it makes `total` and  `docCount` below implicitly `any` type. Secondly, it comes with a fair bit of [overhead](https://medium.com/@mezenok/why-you-should-use-optional-chaining-instead-of-lodash-get-9548e69f7b0e) To maintain type safety, you could potentially use optional chaining instead:

```suggestion
      const total = resp?.rawResponse?.hits?.total;
```